Cannot open "Replay detail" window for replays with "unranked_by_host" validity

**To Reproduce**
1. Open `Replays` tab
2. Click `Adv. search` button
3. Select `Validity` field and `UNRANKED_BY_HOST` value.
4. Click `Search` button.
5. Open any founded replay.

**Log or error message**
```
2022-06-22 10:57:49.220 ERROR 11228 --- [JavaFX Application Thread] c.f.c.exception.GlobalExceptionHandler   : Uncaught exception on Thread[JavaFX Application Thread,5,main]: 

java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1857)
	at javafx.fxml.FXMLLoader$ControllerMethodEventHandler.handle(FXMLLoader.java:1724)
	at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:234)
	at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:191)
	at com.sun.javafx.event.CompositeEventDispatcher.dispatchBubblingEvent(CompositeEventDispatcher.java:59)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:58)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.BasicEventDispatcher.dispatchEvent(BasicEventDispatcher.java:56)
	at com.sun.javafx.event.EventDispatchChainImpl.dispatchEvent(EventDispatchChainImpl.java:114)
	at com.sun.javafx.event.EventUtil.fireEventImpl(EventUtil.java:74)
	at com.sun.javafx.event.EventUtil.fireEvent(EventUtil.java:54)
	at javafx.event.Event.fireEvent(Event.java:198)
	at javafx.scene.Scene$ClickGenerator.postProcess(Scene.java:3586)
	at javafx.scene.Scene$MouseHandler.process(Scene.java:3890)
	at javafx.scene.Scene.processMouseEvent(Scene.java:1874)
	at javafx.scene.Scene$ScenePeerListener.mouseEvent(Scene.java:2607)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:411)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler$MouseEventNotification.run(GlassViewEventHandler.java:301)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$2(GlassViewEventHandler.java:450)
	at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:424)
	at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:449)
	at com.sun.glass.ui.View.handleMouseEvent(View.java:551)
	at com.sun.glass.ui.View.notifyMouse(View.java:937)
	at com.sun.jna.Native.invokeLong(Native Method)
	at com.sun.jna.Function.invoke(Function.java:427)
	at com.sun.jna.Function.invoke(Function.java:360)
	at com.sun.jna.Library$Handler.invoke(Library.java:244)
	at jdk.proxy2/jdk.proxy2.$Proxy91.CallWindowProc(Unknown Source)
	at ch.micheljung.fxwindow.DecorationWindowProcedure.callback(DecorationWindowProcedure.java:192)
	at jdk.internal.reflect.GeneratedMethodAccessor63.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.sun.jna.CallbackReference$DefaultCallbackProxy.invokeCallback(CallbackReference.java:520)
	at com.sun.jna.CallbackReference$DefaultCallbackProxy.callback(CallbackReference.java:551)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$runLoop$3(WinApplication.java:184)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.GeneratedMethodAccessor271.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.sun.javafx.reflect.Trampoline.invoke(MethodUtil.java:77)
	at jdk.internal.reflect.GeneratedMethodAccessor108.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at com.sun.javafx.reflect.MethodUtil.invoke(MethodUtil.java:275)
	at com.sun.javafx.fxml.MethodHelper.invoke(MethodHelper.java:84)
	at javafx.fxml.FXMLLoader$MethodHandler.invoke(FXMLLoader.java:1854)
	... 48 common frames omitted
Caused by: org.springframework.context.NoSuchMessageException: No message found under code 'game.reasonNotValid.unrankedByHost' for locale 'en_US'.
	at org.springframework.context.support.AbstractMessageSource.getMessage(AbstractMessageSource.java:161)
	at com.faforever.client.i18n.I18n.get(I18n.java:102)
	at com.faforever.client.i18n.I18n.get(I18n.java:94)
	at com.faforever.client.replay.ReplayDetailController.configureRatingControls(ReplayDetailController.java:424)
	at com.faforever.client.replay.ReplayDetailController.populateTeamsContainer(ReplayDetailController.java:368)
	at com.faforever.client.replay.ReplayDetailController.setReplay(ReplayDetailController.java:294)
	at com.faforever.client.replay.OnlineReplayVaultController.onDisplayDetails(OnlineReplayVaultController.java:66)
	at com.faforever.client.replay.ReplayCardController.onShowReplayDetail(ReplayCardController.java:172)
	... 58 common frames omitted
````

**Expected behavior**
`Replay Detail` window is open

**Screenshots**
No need

**OS**
Windows

**Client Version**
2022.6.0-exp-2

--- a/src/main/resources/i18n/messages.properties
+++ b/src/main/resources/i18n/messages.properties
@@ -940,6 +940,7 @@ game.reasonNotValid.unitRestriction = Unit Restrictions
 game.reasonNotValid.unrankedMap = Unranked Map
 game.reasonNotValid.short = Game Too Short
 game.reasonNotValid.unrankedMod = Unranked Mod
+game.reasonNotValid.unrankedByHost = Unranked By Host
 game.reasonNotValid.coop = Coop Game
 game.reasonNotValid.draw = Mutual Draw
 game.reasonNotValid.singlePlayer = Single Player

